docs/getting-started/smart-git-sync.md

```diff
@@ -0,0 +1,103 @@
+
+1. You make changes in your Amplication project. This could range from adding a new resource to modifying an existing one.
+2. When you're ready, you perform a new build on the project.
+3. Smart Git Sync springs into action. It identifies all the files that have been altered since the last build.
+4. It then creates a commit on the `amplication` branch, including all these changes.
+
+For example, let's say you've just added a new _Task_ entity to your project. Once you build the project, Smart Git Sync will create a commit on the `amplication` branch that includes all the files related to the _Task_ entity.
+
+This method ensures that every single change you make in your Amplication project is automatically tracked and committed to the `amplication` branch in your Git repository. You can then easily review and merge these changes through pull requests.
+
+:::important
+Do not delete the `amplication` branch from your repository. Deleting this branch will trigger its recreation, and the following pull request will result in a merge conflict. After resolving the merge conflict in the first pull request, you can proceed normally, as subsequent PRs won't create conflicts in the same area.
+:::
+
+![](./assets/building-new-versions/new-build.png)
+
+## The `amplication` branch
+
+Amplication integrates with your repository through a single long-lived branch named `amplication`. This branch serves as the target location for all commits made by Amplication. The creation of a new resource or a change to an existing one will trigger a corresponding commit containing these changes.
+
+Users can merge Amplication-made changes by merging pull requests created by Amplication from the `amplication` branch to the user repository's default branch.
+
+The SGS supports three cases for creating an 'amplication' branch in your repository:
+
+1. **Create Repository**: When a user creates a repository, an 'amplication' branch is made from the repository's `default` branch.
+
+2. **Select Repository**: The existence of an 'amplication' branch is checked. If the branch exists, it's reset, and a new 'amplication' branch is created from the repository's default branch. If the branch doesn't exist, an 'amplication' branch is created from the repository's default branch.
+
+3. **Existed Repository/Deleted Branch**: If a commit is made and the repository lacks an 'amplication' branch, the 'amplication' branch is created from the repository's default branch.
+
+:::important
+Avoid modifying the `amplication` branch. This includes both creating new commits and merging changes from other branches into the 'amplication' branch. Any such action may cause bugs and behavioral inconsistencies, as Amplication will not override or modify the same areas that you changed. To fix this, you will need to delete the 'amplication' branch and resolve the merge conflict in the next PR.
+:::
+
+![](./assets/service-entities-roles-permissions/github-pull-request.png)
```

Comment:
lets use an image with shorter text. the current image shows a very long text that is very hard to understand what it means.